tests/blackbox/stratis_cli_cert.py

```diff
@@ -154,23 +155,29 @@ def test_pool_create_same_name(self):
         Test creating a pool that already exists.
         """
         self.unittest_command(
-            [STRATIS_CLI, "pool", "create", make_test_pool(), DISKS[0]], 1, False, True
+            [STRATIS_CLI, "pool", "create", make_test_pool(DISKS[0]), DISKS[1]],
+            0,
```

Comment:
Typo here...compare with the previous code and you'll see that your expectations of the results of the test have changed, which shouldn't happen.